StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodParams.swift

```diff
@@ -694,7 +714,14 @@ public class STPPaymentMethodParams: NSObject, STPFormEncodable {
             self.affirm = STPPaymentMethodAffirmParams()
         case .swish:
             self.swish = STPPaymentMethodSwishParams()
-        case .paynow, .zip, .amazonPay, .alma, .mobilePay, .konbini, .promptPay, .twint:
+        case .amazonPay:
+            self.amazonPay = STPPaymentMethodAmazonPayParams()
+        case .alma:
+            self.alma = STPPaymentMethodAlmaParams()
+        case .multibanco:
+            self.multibanco = STPPaymentMethodMultibancoParams()
+            self.billingDetails = paymentMethod.billingDetails
```

Comment:
`self.billingDetails = paymentMethod.billingDetails` is being set prior to the case statement. this doesn't seem needed

Reply:
Nice catch, didn't see that